requests_toolbelt/utils/dump.py

```diff
@@ -76,11 +76,11 @@ def _dump_request_data(request, prefixes, bytearr, proxy_info=None):
     bytearr.extend(prefix + b'\r\n')
     if request.body:
         if isinstance(request.body, compat.basestring):
-            bytearr.extend(prefix + _coerce_to_bytes(request.body))
+            bytearr.extend(prefix + _coerce_to_bytes(request.body) + b'\r\n')
         else:
             # In the event that the body is a file-like object, let's not try
             # to read everything into memory.
-            bytearr.extend(b'<< Request body is not a string-like type >>')
+            bytearr.extend(b'<< Request body is not a string-like type >>' + b'\r\n')
     bytearr.extend(b'\r\n')
```

Comment:
I feel like that's what this line was intended to do. Can you give me an example of before/after your change here?

Reply:
I'll get this to you asap

Reply:
This is before
![image](https://user-images.githubusercontent.com/14023728/104217157-a894f680-5400-11eb-92f7-38fddb8ab9ff.png)

This is after 
![image](https://user-images.githubusercontent.com/14023728/104217075-869b7400-5400-11eb-833e-bae8849d9240.png)

The code which generated this was 
```import requests
from requests_toolbelt.utils import dump


def main():
    response = requests.get(
        "http://httpbin.org",
        data="This is some test data that appears directly above the response's status code",
    )
    print(
        dump.dump_response(response, request_prefix=b"", response_prefix=b"").decode()
    )
    # print(response)


main()```

Reply:
Actually, instead of updating those two lines, why not add an extra `bytes.append(b'\r\n')` inside the `if request.body` block

Reply:
Would that be after the `if isinstance ... else` block, but still inside the `if request.body` block? I'll just squash the commits together once we're happy with it

Reply:
Yes that's where I meant I'd like it. Once you squash the commits we can merge this I think

Reply:
Squashed. Merge whenever, thanks for your help! 😊